.gitignore

```diff
@@ -3,3 +3,4 @@
 *.rs.rustfmt
 target/
 Cargo.lock
+.idea/
```

Comment:
Okay, in that case, let's create a section in the gitignore file with a descriptive comment and put it there. You're making lots of contribution and there's no reason to slow it down with such a small thing.